src/Infrastructure/Form/Regulation/GeneralInfoFormType.php

```diff
@@ -102,7 +107,7 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
                     'label' => null,
                     'prototype_name' => '__visa_name__',
                     'entry_options' => [
-                        'label' => 'regulation.general_info.visa',
+                        'label' => 'regulation.general_info.visa.sample',
```

Comment:
Plutôt ça pour mieux s'y retrouver aussi ?

```suggestion
                        'label' => 'regulation.general_info.additional_visas.entry',
```